### DISCUSSION TITLE
[Question] Add Clover Support?

### DISCUSSION THREAD

**Discussion by plsh2me** [badstorm/surface-pro-7-opencore#37] [Q&A]
@badstorm @jackjack1-su  Hi, I am requesting you guys to release at least one clover bootloader version EFI folder. 
As you have a detailed idea of this device driver patches, you may help me to get the clover folder.
Thank you

**Comment by jackjack1-su** [comment]
Nope.

**Comment by jackjack1-su** [comment]
There's no reason to support clover.

**Comment by plsh2me** [comment]
I have converted all the settings from opencore to clover config, but the system not boot with this. Please help me to set the proper boot arg for clover.config. 
[config.zip](https://github.com/badstorm/surface-pro-7-opencore/files/6796063/config.zip)

Thank you

**Comment by badstorm** [comment]
Why you need a Clover version? What Clover give that OpenCore dosen't? The intent of this repo is supporting OpenCore since it give a modern approch of hackintosh. Sincerly there is no interest to support Clover but I leave this issue open for some weeks, so if there is some user interested in helping you you can discuss here. If no one response, I'll close it.

**Comment by badstorm** [comment]
I moved the question in "Discussions" so we can leave it open!

**Comment by jackjack1-su** [comment]
Maybe you can "redo" the guide of converting clover to opencore.
I think it will work.
